Expensify/App
# Add reset bank account modal

### Details

Adds the ability to delete a bank account in setup so users can restart the VBA flow.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/180264

### Tests / QA Steps
1. Create a bank account in setup on NewDot by creating a workspace and following the instructions to create an OPEN bank account [here](https://stackoverflow.com/c/expensify/questions/342)
2. Stop at the Personal Information step and exit the flow
3. Tap "Connect bank account" in the workspace page
4. Tap "Start over"
5. Verify a modal appears with the following content
![2021-10-13_11-20-02](https://user-images.githubusercontent.com/32969087/137214134-c191fd00-946e-4767-8c5e-c8d654d56255.png)
6. Select "Cancel" and verify the modal closes
7. Press "Start over" again and this time select "Yes, start over"
8. Verify you are brought back to the "Streamline payments" page
9. Begin setting up a bank account again but use the VERIFIYING instructions
10. Go all the way to the end and verify you see the "No, let's start over" button
11. Tap and verify the modal works as before. Reset the account.
12. Add an account in the OPEN state again going all the way to the end of the flow
13. Verify on the `/enable` step that a "Disconnect bank account" option is available
![2021-10-13_11-44-30](https://user-images.githubusercontent.com/32969087/137216883-7a7e1240-5069-4a56-9095-b5393df75bc9.png)
14. Tap this option and verify a modal open
![2021-10-13_11-50-40](https://user-images.githubusercontent.com/32969087/137217485-93a64f6e-3458-428f-855c-9bbd9cebd7fc.png)
15. Select "Yes, disconnect my bank account" and everything is reset

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
